### DISCUSSION TITLE
ECC [0-8] mapping to L/M/Q/H

### DISCUSSION THREAD

**Discussion by mbaros100** [zxing-cpp/zxing-cpp#447] [Q&A] [answered]
Hello,

Can anyone please tell me if there is a correspondence between ecc levels [0-8] and [L/M/Q/H]?

Thanks a lot.

**Comment by axxel** [comment, marked as answer]
I guess you are talking about the `MultiformatWriter::setEccLevel` API?
 * 1,2 -> L
 * 3,4 -> M
 * 5,6 -> Q
 * 7,8 -> H

See https://github.com/zxing-cpp/zxing-cpp/blob/f323f1ebea701ef351a14eacb2dccd5284d57184/core/src/MultiFormatWriter.cpp#L41

**Reply by mbaros100** [reply]
yes. thanks a lot
